lib/sierra-models/base.js

```diff
@@ -73,25 +73,19 @@ class SierraBase {
           //    add parallel to the primary
           field.parallel = { ...parallelsLookup[subfieldSixSuffix] }
           //    set value to null, so we know that parallel has been matched
-          parallelsLookup[subfieldSixSuffix] = null
```

Comment:
Proposing just removing the entry from the hash entirely so we don't have to filter on nulls later

Reply:
this was my original idea but i swapped it out for some reason i no longer recall...